contrib/JuliaConfig.jl

```diff
@@ -0,0 +1,69 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+module JuliaConfig
+
+export cflags, ldflags, ldlibs, allflags
+
+threadingOn() = ccall(:jl_threading_enabled, Cint, ()) != 0
```

Comment:
We should probably take this opportunity to use the predominant naming conventions, which means that names should be lowercase or lower snake case rather than camel case.

Reply:
@ararslan: done